src/components/controls/dates/DatePicker/DatePicker.stories.tsx

```diff
@@ -92,6 +92,20 @@ ErrorMessage.args = {
   status: "fail",
 }
 
+export const AllowFreeInput = Template.bind({})
+AllowFreeInput.args = {
+  label: "Label",
+  placeholder: "Placeholder",
+  allowFreeInput: true,
+}
```

Comment:
What about extending `WithLabel` here as we do in the other stories?

```suggestion
AllowFreeInput.args = {
  ...WithLabel.args,
  allowFreeInput: true,
}
```